<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HexEditorPage</class>
 <widget class="QWidget" name="HexEditorPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>721</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>&lt;b&gt;Configure Hex Editor&lt;/b&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line14">
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="readOnlyCheckBox">
       <property name="toolTip">
        <string>Select whether files shall be opened in read only mode</string>
       </property>
       <property name="text">
        <string>Open files 'read only'</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="overwriteCheckBox">
       <property name="toolTip">
        <string>Select whether the editor shall be started in Overwrite mode</string>
       </property>
       <property name="text">
        <string>Overwrite data</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Address Area</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="4">
       <widget class="QCheckBox" name="addressAreaCheckBox">
        <property name="toolTip">
         <string>Select whether the address area shall be shown</string>
        </property>
        <property name="text">
         <string>Show Address Area</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <layout class="QHBoxLayout">
        <item>
         <widget class="QLabel" name="textLabel1_20">
          <property name="text">
           <string>Address Area Width:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="addressAreaWidthSpinBox">
          <property name="toolTip">
           <string>Enter the width of the address area in characters</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="suffix">
           <string> Chars</string>
          </property>
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>12</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="TextLabel1_3_4">
        <property name="text">
         <string>Foreground:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="addressAreaForeGroundButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Select the foreground colour of the address area</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="TextLabel1_3_2_4">
        <property name="text">
         <string>Background:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="addressAreaBackGroundButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Select the background colour of the address area</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>ASCII Area</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QCheckBox" name="asciiAreaCheckBox">
        <property name="toolTip">
         <string>Select whether the ASCII area shall be shown</string>
        </property>
        <property name="text">
         <string>Show ASCII Area</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Highlighting</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="4">
       <widget class="QCheckBox" name="highlightingCheckBox">
        <property name="toolTip">
         <string>Select whether changed data shall be highlighted</string>
        </property>
        <property name="text">
         <string>Highlight Changed Data</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="TextLabel1_3_2">
        <property name="text">
         <string>Foreground:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="highlightingForeGroundButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Select the foreground colour for highlighted data</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="TextLabel1_3_2_2">
        <property name="text">
         <string>Background:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="highlightingBackGroundButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Select the background colour for highlighted data</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Selection</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="TextLabel1_3_3">
        <property name="text">
         <string>Foreground:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectionForeGroundButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Select the foreground colour of the selection</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="TextLabel1_3_2_3">
        <property name="text">
         <string>Background:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="selectionBackGroundButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Select the background colour of the selection</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Font</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="monospacedFontButton">
        <property name="toolTip">
         <string>Press to select the font to be used (this must be a monospaced font)</string>
        </property>
        <property name="text">
         <string>Monospaced Font</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="monospacedFontSample">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string notr="true">01 23 45 67 89 ab cd ef 70 81 92 a3 b4 c5 d6 e7 f9</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="title">
      <string>Recent Files</string>
     </property>
     <layout class="QHBoxLayout" name="_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Number of recent files:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="recentFilesSpinBox">
        <property name="toolTip">
         <string>Enter the number of recent files to remember</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>readOnlyCheckBox</tabstop>
  <tabstop>overwriteCheckBox</tabstop>
  <tabstop>addressAreaCheckBox</tabstop>
  <tabstop>addressAreaWidthSpinBox</tabstop>
  <tabstop>addressAreaForeGroundButton</tabstop>
  <tabstop>addressAreaBackGroundButton</tabstop>
  <tabstop>asciiAreaCheckBox</tabstop>
  <tabstop>highlightingCheckBox</tabstop>
  <tabstop>highlightingForeGroundButton</tabstop>
  <tabstop>highlightingBackGroundButton</tabstop>
  <tabstop>selectionForeGroundButton</tabstop>
  <tabstop>selectionBackGroundButton</tabstop>
  <tabstop>monospacedFontButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
